mapserver/docker-compose.yml

```diff
@@ -4,23 +4,53 @@ services:
     image: camptocamp/mapserver:7.4
     restart: unless-stopped
 
+  rabbitmq:
```

Comment:
Do not use rabbitmq for now.
Use in memory broker for celery.
This is because we want to stay as lightweight as possible for the generic case.
GeoCRIS deployment would not allow us to freely install more service.